docs/concepts/authentication_permissions.rst

```diff
@@ -0,0 +1,58 @@
+.. _authentication-permissions:
+
+Authentication and Permissions
+######################################
+
+Users access Aspects data and reports by signing into Superset, which will confirm their identity
+and permissions with the LMS. This is done using Single Sign On (SSO) and JSON Web Tokens (JWT).
+
+By default Superset provides the following roles:
+
+- Alpha
+- Gamma
+- granter
+- Public
+- sql_lab
+
+You can find more information about these roles in the 
+`Superset documentation <https://superset.apache.org/docs/security/#roles>`_.
+
+We do not use any of the provided roles, instead we have created custom roles for our specific
```

Comment:
I believe we use `Alpha` role for admins and the `Gamma` role for instructors / course staff due to the way Superset does things.

Reply:
In fact, we copied the `Gamma` role permissions and transferred them to the `Instructor` role and remove the permissions we considered not proper of an `Instructor`

Reply:
Right, but we do still use them? https://github.com/openedx/tutor-contrib-aspects/blob/a3352be757386dfe2425b9c60f7c128a12ab6c30/tutoraspects/templates/aspects/apps/superset/pythonpath/superset_config_docker.py#L61

Reply:
I didn't remove I think it still covers the data-analysis use case, but it's not used anywhere
https://github.com/search?q=repo%3Aopenedx%2Ftutor-contrib-aspects%20gamma&type=code